pkg/controllers/capiinstaller/capi_installer_controller.go

```diff
@@ -277,14 +277,14 @@ func (r *CapiInstallerController) setAvailableCondition(ctx context.Context, log
 
 	conds := []configv1.ClusterOperatorStatusCondition{
 		operatorstatus.NewClusterOperatorStatusCondition(capiInstallerControllerAvailableCondition, configv1.ConditionTrue, operatorstatus.ReasonAsExpected,
-			"CAPI Installer Controller works as expected"),
+			"Cluster API Installer Controller works as expected"),
```

Comment:
All these `CAPI Installer Controller` should stay as is, as that's the actual controller name.